lib/stat_tracker.rb

```diff
@@ -127,12 +127,33 @@ def lowest_scoring_home_team
 
   # Season Statistics
 
-  def winningest_coach
-
+  def winningest_coach(season_id)
+    games_in_season = @game_teams.find_all do |game|
+      game.game_id[0..3] == season_id[0..3]
+    end
```

Comment:
This is just a note that we'll want to update this portion to match the helper method created on another branch -> .season_games(season_id)